mne/evoked.py

```diff
@@ -377,7 +377,7 @@ def plot(self, picks=None, exclude='bads', unit=True, show=True, ylim=None,
         show : bool
             Call pyplot.show() at the end or not.
         ylim : dict
-            ylim for plots. e.g. ylim = dict(eeg=[-200e-6, 200e-6])
+            ylim for plots. e.g. ylim = dict(eeg=[-200, 200])
```

Comment:
I can't try it for myself right now, but ERPs are  almost always in the single-digit uV range - that would mean, either 200e-6 if the data is in V, or somewhere between 2 and 20 if it corresponds to the ylim after scaling the data. I can check later.